velox/exec/tests/utils/AssertQueryBuilder.cpp

```diff
@@ -155,6 +155,27 @@ std::shared_ptr<Task> AssertQueryBuilder::assertResults(
   return cursor->task();
 }
 
+RowVectorPtr AssertQueryBuilder::copyResults(memory::MemoryPool* pool) {
```

Comment:
This is likely from another PR , maybe rebase.

Reply:
This PR depends on 2 other PRs. One just landed, so rebased. The other one is still open: #1661